From: Jon Ludlam <jonathan.ludlam@eu.citrix.com>
Date: Fri, 7 Oct 2011 14:44:33 +0100
Subject: dont-build-xiu

---
 OMakefile           |    2 --
 ocaml/xiu/OMakefile |    4 ----
 2 files changed, 0 insertions(+), 6 deletions(-)

--- a/OMakefile
+++ b/OMakefile
@@ -163,8 +163,6 @@
 	ocaml/xenops/cancel_utils_test \
 	ocaml/xenguest/xenguest \
 	ocaml/xenguest/dumpcore \
-	ocaml/xiu/libxenctrl_xiu.so \
-	ocaml/xiu/xiu \
 	ocaml/xapi/quicktestbin \
 	ocaml/xapi/sparse_dd \
 	ocaml/xapi/storage_impl_test \
--- a/ocaml/xiu/OMakefile
+++ b/ocaml/xiu/OMakefile
@@ -19,13 +19,9 @@
 .PHONY: install
 install:
 	mkdir -p $(DESTDIR)$(LIBEXECDIR)
-	$(IPROG) xiu $(DESTDIR)$(LIBEXECDIR)/
 	mkdir -p $(DESTDIR)$(OPTDIR)/lib
-	$(IDATA) libxenctrl_xiu.so $(DESTDIR)$(OPTDIR)/lib
 
 .PHONY: sdk-install
 sdk-install:
 	mkdir -p $(DESTDIR)$(LIBEXECDIR)
-	$(IPROG) xiu $(DESTDIR)$(LIBEXECDIR)/
 	mkdir -p $(DESTDIR)$(OPTDIR)/lib
-	$(IDATA) libxenctrl_xiu.so $(DESTDIR)$(OPTDIR)/lib
