From: Jon Ludlam <jonathan.ludlam@eu.citrix.com>
Date: Fri, 7 Oct 2011 14:44:39 +0100
Subject: remove-missing-xc-calls

---
 ocaml/xenops/domain.ml |   23 ++++++++++++-----------
 1 files changed, 12 insertions(+), 11 deletions(-)

--- a/ocaml/xenops/domain.ml
+++ b/ocaml/xenops/domain.ml
@@ -79,7 +79,7 @@
 
 let save_signature = "XenSavedDomain\n"
 let qemu_save_signature = "QemuDeviceModelRecord\n"
-let hvmloader = "/usr/lib/xen/boot/hvmloader"
+let hvmloader = "/usr/lib/xen-4.1/boot/hvmloader"
 let releaseDomain = "@releaseDomain"
 let introduceDomain = "@introduceDomain"
 
